Examples/Algorithms/Vertexing/src/VertexFitterAlgorithm.cpp

```diff
@@ -40,7 +40,7 @@ ActsExamples::VertexFitterAlgorithm::VertexFitterAlgorithm(
 ActsExamples::ProcessCode ActsExamples::VertexFitterAlgorithm::execute(
     const ActsExamples::AlgorithmContext& ctx) const {
   using MagneticField = Acts::ConstantBField;
-  using Stepper = Acts::EigenStepper<MagneticField>;
+  using Stepper = Acts::EigenStepper<>;
```

Comment:
Similarly here, @baschlag  - after changing the bfield not to be a template parameter anymore, one could actually stay mostly with the original classname w/o redefining.

Another question: shouldn't that be using the magnetic field as configured by the `Options` and not force-used the constant field?

Reply:
Hm. Good point. It was hard-coded to `ConstantBField` before, and this PR doesn't change this fact.

About the aliases: exactly! We should do another pass to remove them where they're not needed.

Reply:
I will change it after this PR went it.

Reply:
BTW: The reason it was hard-coded was that the entire vertexing was implicitly templated on the bfield type. So there was no easy way to make the bfield type configurable. Now with this PR we can do so (which is nice!)